src/scene/lightmapper/bake-light-ambient.js

```diff
@@ -22,6 +22,7 @@ class BakeLightAmbient extends BakeLight {
             normalOffsetBias: 0.05,
             shadowBias: 0.2,
             shadowDistance: 1,  // this is updated during shadow map rendering
+            shadowIntensity: 1,
```

Comment:
we should not need this, as that is the default value (just making sure the default value works)